docs/csharp/language-reference/operators/patterns.md

```diff
@@ -1,41 +1,66 @@
 ---
 title: "Patterns - C# reference"
-description: "Learn about the patterns supported by C# switch and is expressions and C# switch statement - C# reference"
-ms.date: 03/31/2021
+description: "Learn about the patterns supported by C# pattern matching expressions and statements - C# reference"
+ms.date: 04/05/2021
+f1_keywords: 
+  - "and_CSharpKeyword"
+  - "or_CSharpKeyword"
+  - "not_CSharpKeyword"
 helpviewer_keywords: 
   - "pattern matching [C#]"
+  - "and keyword [C#]"
+  - "or keyword [C#]"
+  - "not keyword [C#]"
 ---
 # Patterns (C# reference)
 
-Intro.
+Beginning with C# 7.0, the following C# expressions and statements support pattern matching:
```

Comment:
I would add a sentence noting that pattern matching syntax has been enhanced in each version of C#:

```suggestion
C# introduced pattern matching in C# 7.0. Pattern matching expressions have been extended in each version since 7.0 The following C# expressions and statements support pattern matching:
```

Reply:
@BillWagner thank you; I've rephrased your suggestion; please check the latest commit.